scalafix-core/src/main/scala/scalafix/rewrite/ExplicitUnit.scala

```diff
@@ -5,6 +5,7 @@ import scala.meta._
 
 case object ExplicitUnit extends Rewrite {
   override def rewrite(ctx: RewriteCtx): Patch = {
+    org.scalameta.logger.elem(ctx.tree)
```

Comment:
@olafurpg This was introduced in your previous commit. Do you want me to get rid of this?